### ISSUE TITLE
Editing xopp files doesn't work on windows

### ISSUE THREAD

**Issue by Nereg** [gabrielrueda/obsidian-xournal-integration-plugin#5]
Hi! I love your plugin, but the "Edit drawing" command doesn't work. It says that it can't execute a command:
![image](https://github.com/gabrielrueda/obsidian-xournal-integration-plugin/assets/47698366/7a433001-3dba-4da5-8d1c-a28e8b4fa9d1)
I can execute the `xournalpp` executable from anywhere (I have it in my PATH), so that is fine. And, if I type the same command in my shell, it works, but no xournal windows pop up. Maybe that's because of the `;` between the two commands ? But the `-p` flag doesn't seem to work in xournalpp in windows at all (I tried to export to pdf before)
Also, I can share pre-build files, if you want. Built from the main branch with node 20.

**Comment by gabrielrueda** [comment]
@Nereg 
I have looked into this a bit. I'm pretty sure the issue is that the xournal cli is not supported on windows. Which means you cannot open xournal files and export as pdf with command line arguments. 

For us to get windows support, we would have to find a workaround. 

You can try to remove the exporting to PDF part and see if that does something:

1. Go to File:
Modals/edit_drawing_modal.ts

2. Change the line:
const command = "xournalpp " + filePath + "; xournalpp -p " + pdfFilePath + " " + filePath;

to 

const command = "xournalpp " + filePath ;

3. Let me know if this part is successful or not.

**Comment by Nereg** [comment]
Yes, the opening part works with the edit you suggested. I solved the export problem with a hacky workaround using a xournal++ plugin which exports and then saves the file in a single step. Also made a keybind for it, sadly, not the Ctrl+S.
